brownie-config.yaml

```diff
@@ -46,6 +46,7 @@ compiler:
             runs: 200
         remappings: 
             - "@openzeppelin-4.3.2=./node_modules/openzeppelin-4.3.2"
+            - "@openzeppelin-4.7.0=./node_modules/openzeppelin-4.7.0"
```

Comment:
if we include this version. probably we can retire the 4.3.2, it has to compile since it supposed to be backward compatible

Reply:
I can work on this.